Dockerfile

```diff
@@ -1,4 +1,4 @@
-FROM owasp/zap2docker-stable as base
+FROM owasp/zap2docker-weekly:w2022-05-16 as base
```

Comment:
are we going to be using an specific tag? or should it be just `FROM owasp/zap2docker-weekly`  ?

Reply:
Using a specific tag so we can test with that since the weekly's aren't guaranteed to be stable. I'll double check at stand-up that we want to go that route though.